utility_scripts/slurm_run_vep.sh

```diff
@@ -138,52 +136,46 @@ validateCommandLine() {
 }
 
 runVep() {
-  local args=()
-  args+=("exec")
-  args+=("${vep_image}")
-  args+=("vep")
-  args+=("--input_file" "${input}")
-  args+=("--format" "vcf")
-  args+=("--output_file" "${output}")
-  args+=("--vcf")
-  args+=("--compress_output" "gzip")
-  args+=("--sift" "s")
-  args+=("--polyphen" "s")
-  args+=("--numbers")
-  args+=("--symbol")
-  args+=("--shift_3prime" "1")
-  args+=("--allele_number")
-  args+=("--refseq")
-  args+=("--total_length")
-  args+=("--no_stats")
-  args+=("--offline")
-  args+=("--cache")
-  args+=("--dir_cache" "${resources_directory}/vep/cache")
-  args+=("--species" "homo_sapiens")
-  args+=("--assembly" "${ASSEMBLY}")
-  args+=("--fork" "4")
-  args+=("--dont_skip")
-  args+=("--allow_non_variant")
-  args+=("--use_given_ref")
-  args+=("--exclude_predicted")
-  args+=("--flag_pick_allele")
-  args+=("--plugin" "Grantham")
-  if [[ "${PG}" == true ]]
-  then
-    args+=("--per_gene")
-  fi
-  args+=("--dir_plugins" "${resources_directory}/vep/plugins")
-
-  if [[ "${ASSEMBLY}" == "GRCh37" ]]
-  then
-    args+=("--plugin" "SpliceAI,snv=${resources_directory}GRCh37/spliceai_scores.masked.snv.hg19.vcf.gz,indel=${resources_directory}GRCh37/spliceai_scores.masked.indel.hg19.vcf.gz")
-    args+=("--custom" "${resources_directory}GRCh37/gnomad.total.r2.1.1.sites.stripped.patch1.vcf.gz,gnomAD,vcf,exact,0,AF,HN")
-    args+=("--custom" "${resources_directory}GRCh37/hg19.100way.phyloP100way.bw,phyloP,bigwig,exact,0")
-  else
+    local args=()
+    args+=("exec")
+    args+=("${vep_image}")
+    args+=("vep")
+    args+=("--input_file" "${input}")
+    args+=("--format" "vcf")
+    args+=("--output_file" "${output}")
+    args+=("--vcf")
+    args+=("--compress_output" "gzip")
+    args+=("--no_stats")
+    args+=("--offline")
+    args+=("--cache")
+    args+=("--dir_cache" "${resources_directory}/vep/cache")
+    args+=("--species" "homo_sapiens")
+    args+=("--assembly" "GRCh38")
+    args+=("--refseq")
+    args+=("--exclude_predicted")
+    args+=("--use_given_ref")
+    args+=("--symbol")
+    args+=("--flag_pick_allele")
+    args+=("--sift" "s")
+    args+=("--polyphen" "s")
+    args+=("--total_length")
+    args+=("--shift_3prime" "1")
+    args+=("--allele_number")
+    args+=("--numbers")
+    args+=("--dont_skip")
+    args+=("--allow_non_variant")
+    args+=("--fork" "4")
+    args+=("--dir_plugins" "${resources_directory}/vep/plugins")
+    args+=("--plugin" "Grantham")
+    args+=("--safe")
     args+=("--plugin" "SpliceAI,snv=${resources_directory}GRCh38/spliceai_scores.masked.snv.hg38.vcf.gz,indel=${resources_directory}GRCh38/spliceai_scores.masked.indel.hg38.vcf.gz")
```

Comment:
resources_directory sometimes superseded by slash, sometimes not:
```
${resources_directory}/vep/plugins
${resources_directory}GRCh38
```
suggestion, always add `/`, remove `/` from default value.